apps/test/unit/componentLibrary/TooltipTest.jsx

```diff
@@ -0,0 +1,28 @@
+import {render, screen} from '@testing-library/react';
+import React from 'react';
+
+import {WithTooltip} from '@cdo/apps/componentLibrary/tooltip';
+
+import {expect} from '../../util/reconfiguredChai';
+
+describe('Design System - Tooltip', () => {
+  it('Tooltip - renders with correct label', () => {
```

Comment:
I wanted to check css visibility attribute value here, but I didn't succed because of our React testing library + reconfiguredChai tests combination.
If anyone has any ideas how we can do this - will be super thankfull. If no - thinking about merging as is or remove test file completely.

Here's what I've tried, but it didn't helped (at least part of it, I also added data-testId to Tooltip for that tests cause you can see getByTestId here):
```
it('Tooltip - renders with correct label', () => {
  const WithTooltipToRender = () => (
    <WithTooltip tooltipProps={{tooltipId: 'tooltip1', text: 'tooltipText'}}>
      <button>hover me</button>
    </WithTooltip>
  );
  const {rerender} = render(<WithTooltipToRender />);

  let tooltip = screen.getByText('tooltipText');
  let tooltipContainer = screen.getByTestId('tooltip1');
  let tooltipTrigger = screen.getByText('hover me');

  expect(tooltip).to.exist;
  expect(tooltipContainer).to.exist;
  // expect(tooltip).not.to.be.visible;
  expect(tooltipContainer.style.getPropertyValue('visibility')).to.equal(
    'hidden'
  );
  // expect(tooltip.style.visibility).to.equal('hidden');
  // expect(tooltip).to.have.('visibility', 'hidden');
  // expect(tooltip.getCssProperty('visibility').value).to.be('hidden');
  expect(tooltipTrigger).to.exist;

  userEvent.hover(tooltipTrigger);

  rerender(<WithTooltipToRender />);

  tooltip = screen.getByText('tooltipText');
  tooltipContainer = screen.getByTestId('tooltip1');
  expect(tooltip).to.exist;
  expect(tooltipContainer.style.visibility).to.equal('visible');
  // expect(tooltip.style.visibility).to.equal('visible');

  // expect(tooltip).not.to.be.visible;
  //
  // expect(tooltip).to.be.visible;

  // expect(toggle).to.exist;
  // expect(screen.getByText(toggleLabel)).to.exist;
})
```